README.md

```diff
@@ -22,6 +22,8 @@ A unity platforming game, featuring ugprades and differing levels. Made by Ediso
 1. Follow steps 1-4 in [playing](https://github.com/justapotato213/Jump-of-Fate/edit/main/README.md#playing). 
 2. Open [LevelGenerator/imageGenerator.py](LevelGenerator/imageGenerator.py).
 3. Edit the code (imageGenerator.py) using an IDE of your choice. 
+4. Add slices in the form of .jpg images inside (Input_Images) folder.
```

Comment:
if your trying to do link things you have to do it like this: 
`[words](link) `
for this one it would be 
`[Input_Images](LevelGenerator/Input_Images)`

Reply:
Also github allows paths like that, it'll just reference the thing inside the github so do that